modules/account-lib/src/coin/sol/transaction.ts

```diff
@@ -30,6 +30,10 @@ export class Transaction extends BaseTransaction {
     this._solTransaction = tx;
   }
 
+  private get numSignaturesRequired(): number {
```

Comment:
I think here we need to follow the bitgo standard for this. Here is related to number of signatures.
https://docs.google.com/document/d/1cMGyXa8GwgwDMcZ2Uv7dMDsqRWlIEp1MRdVgpPW8p1o/edit#bookmark=id.u64ygc8x58e5

Reply:
good catch, renamed